app/src/types/user.ts

```diff
@@ -79,3 +79,8 @@ export type UsersResponse = {
   createdAt: Date;
   emailVerified: boolean;
 };
+
+export type SignupResponse = {
+  token: string;
```

Comment:
probably userId ?

should match with https://github.com/dltxio/proxy.idem.com.au/pull/60/files